frontend/src/components/common/checkbox/checkbox.tsx

```diff
@@ -1,38 +1,35 @@
 import * as React from 'react';
 import clsx from 'clsx';
-import { useController, Control } from 'react-hook-form';
-import { FormDefaultValue } from 'common/types';
+import { FormRegisterCb, FormErrors } from 'common/types';
 
 import styles from './styles.module.scss';
 
 interface Props {
   name: string;
+  value?: string;
   label: string;
   isDisabled?: boolean;
-  control: Control;
-  defaultValue?: FormDefaultValue;
+  errors: FormErrors;
+  register: FormRegisterCb;
 }
 
 const Checkbox: React.FC<Props> = ({
   name,
+  value,
   label,
   isDisabled,
-  control,
-  defaultValue,
-}) => {
-  const {
-    field,
-    meta: { invalid },
-  } = useController({ name, control, defaultValue });
-
+  errors,
+  register,
+}) =>{
   return (
-    <label className={clsx(styles.checkboxRow, invalid && styles.error)}>
+    <label className={clsx(styles.checkboxRow, errors[name] && styles.error)}>
```

Comment:
```suggestion
    <label className={clsx(styles.checkboxRow, hasError && styles.error)}>
```
const hasError = Boolean(errors[name]);

We also need to output error using `ErrorMessage`